README.md

```diff
@@ -178,6 +178,7 @@ components, see the [A-Frame Registry](https://aframe.io/aframe-registry/).
 - [Star](https://github.com/andreasplesch/aframe-star-component) - 2D stars
 - [Stereo](https://github.com/oscarmarinmiro/aframe-stereo-component) - Enable rendering different entities for each eye, using three.js layers so you can render 3D stereoscopic videos (full and half-dome)
 - [Stereo Cube](https://github.com/wallabyway/aframe-stereocube) - Use stereoscopic cube-map textures (i.e., left and right eye textures)
+- [Stories](https://afsdemo.aschnapp.com) - Demo of [aframe-react-stories](https://npmjs.com/aframe-react-stories) module. A module to quickly add social media style stories in VR to your project. 
```

Comment:
I would change the URL to `https://www.npmjs.com/package/aframe-react-stories` (since that's what it redirects to

Reply:
thanks for moving and tweaking this.

what do you think of this text?

```
- [Stories](https://afsdemo.aschnapp.com) - Quickly add social media-style stories in VR to your React-based project. [[Demo]](https://www.npmjs.com/package/aframe-react-stories)
```